src/styles.css

```diff
@@ -0,0 +1,1209 @@
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 6px 9px;
+}
+
+.role-dot {
+  width: 10px;
+  height: 10px;
+  background: #c1c7d0;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 6.5px 12px;
+  border-radius: 50%;
+}
+
+.role-job {
+  height: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 4px 0.5px;
+}
+
+.role-year {
+  width: 29px;
+  height: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 4px 0.5px;
+}
+
+.project-description {
+  margin-left: 5%;
+}
+
+.project-description > h1 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+}
+
+.project-details {
+  width: 100%;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-top: 5%;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.skills {
+  display: flex;
+  float: right;
+  align-items: center;
+  position: static;
+  padding: 0.5px;
+  width: 100%;
+  height: 24px;
+  left: 0.5px;
+  top: 180px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  margin-top: 4%;
+}
+
+.skill {
+  position: static;
+  width: 29px;
+  height: 16px;
+  left: 12px;
+  top: 4px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  margin: 0.5px 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.button-click {
+  position: static;
+  width: 70%;
+  height: 72px;
+  left: 0.5px;
+  top: 216px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+}
+
+.button-resume {
+  position: static;
+  width: 70%;
+  height: 72px;
+  left: 0.5px;
+  top: 216px;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  margin: 24px 0.5px;
+  padding-left: 20px;
+}
+
+.btn {
+  flex-direction: row;
+  align-items: center;
+  background: #fff;
+  padding: 7px;
+  top: 12px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  margin-top: 5%;
+}
+
+.btn:hover {
+  background-color: #396df2;
+  color: #fff;
+}
+
+button:active {
+  background-color: #2230d2;
+}
+
+.btn:active {
+  background-color: #2230d2;
+}
+
+.about {
+  background: #fff;
+  width: 70%;
+  padding: 64px;
+  border-top-right-radius: 55px;
+}
```

Comment:
- [ ] Your about section is overflowing and doesn't match the `Figma` design. It's important to `fix` this issues as it going to help make your design look much better and also help to create a good user experience. To fix this issue s consider the below suggestions: 
   - [ ] Remove the set `width` and `padding`. After removing them your design should start looking better already. 
   - [ ] Adjust the spacing around your content to match that of the `Figma` design after removing the mentions style. See below for more details:

<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="299" alt="Screenshot 2023-07-06 at 16 05 32" src="https://github.com/gdvl777/Portfolio-finish-mobile-version/assets/107192076/c80107f8-2139-48de-a64a-f54c368b610d">
</td>
<td>
<img width="339" alt="Screenshot 2023-07-06 at 16 05 01" src="https://github.com/gdvl777/Portfolio-finish-mobile-version/assets/107192076/c55ae418-0633-4f06-bf76-527c61d39db7">
</td>
</tr>
</table>